doc/build-instructions/Build_Instructions_V13.md

```diff
@@ -0,0 +1,651 @@
+- [Windows JDK 12](https://adoptopenjdk.net/releases.html?variant=openjdk12#x64_win), which is used as the boot JDK.
+- [Microsoft Visual Studio 2017]( https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=Community&rel=15), which is the default compiler level used by OpenJDK13.
+- [Freemarker V2.3.8](https://sourceforge.net/projects/freemarker/files/freemarker/2.3.8/freemarker-2.3.8.tar.gz/download)
+- [LLVM/Clang](http://releases.llvm.org/7.0.0/LLVM-7.0.0-win64.exe)
+- [NASM Assembler v2.13.03 or newer](https://www.nasm.us/pub/nasm/releasebuilds/?C=M;O=D)
+
+Add the binary path of Clang to the `PATH` environment variable to override the older version of clang integrated in Cygwin. e.g.
+```
+export PATH="/cygdrive/c/LLVM/bin:$PATH" (in Cygwin)
+```
+
+Add the path to `nasm.exe` to the `PATH` environment variable to override the older version of NASM installed in Cygwin. e.g.
+```
+export PATH="/cygdrive/c/NASM:$PATH" (in Cygwin)
+```
+
+Update your `LIB` and `INCLUDE` environment variables to provide a path to the Windows debugging tools with the following commands:
+
+```
+set INCLUDE=C:\Program Files\Debugging Tools for Windows (x64)\sdk\inc;%INCLUDE%
+set LIB=C:\Program Files\Debugging Tools for Windows (x64)\sdk\lib;%LIB%
+```
+
+   You can download Visual Studio, Freemarker manually or obtain them using the [wget](http://www.gnu.org/software/wget/faq.html#download) utility. If you choose to use `wget`, follow these steps:
+
+- Open a cygwin terminal and change to the `/temp` directory:
+```
+cd /cygdrive/c/temp
+```
+
+- Run the following commands:
+```
+wget https://aka.ms/vs/15/release/vs_community.exe 
+wget https://sourceforge.net/projects/freemarker/files/freemarker/2.3.8/freemarker-2.3.8.tar.gz/download -O freemarker.tgz
+```
+- Before installing Visual Studio, change the permissions on the installation file by running `chmod u+x vs2017.exe`.
+- Install Visual Studio by running the file `vs2017.exe` (There is no special step required for downloading/installing VS2017. Please follow the guide of the downloaded installer to install all required components, especially for VC compiler).
+
+Not all of the shared libraries that are included with Visual Studio are registered during installation.
+In particular, the `msdia140.dll`(VS2017) library must be registered manually.
+To do so, execute the following from a command prompt:
+
+```
+**VS2017**
```

Comment:
I don't think we need this header any more since there is only one version.

Reply:
Removed.